### PR TITLE
Update Readme.md and package.json file

### PR DESCRIPTION
**Pull request by pooja-muchandikar**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,9 +2,13 @@
 A Script to conduct visual testing with [BackstopJS](https://github.com/garris/BackstopJS). This script can give you the confidence of delivering a bug free UI and can save you from monotonus work of regression testing. The Visual Automation framework is a quick and efficient tool to identify visual changes without vetting through the whole application manually. 
 
 
+## Changelog
+## v1.1 [June 7 2023]
+Add sitemap.xml Feature.
```
Comment (juhi123) [inline README.md:8]: @pooja-muchandikar Can you please add this under enhancement

Comment (pooja-muchandikar) [reply README.md:8]: Sure.

Comment (juhi123) [reply README.md:8]: Or I think most appropriate will be `NEW FEATURES`

Comment (pooja-muchandikar) [reply README.md:8]: @juhi123 done

**Review by juhi123 — APPROVED** [review]

**Pull request merged**